### ISSUE TITLE
TWW Update

### ISSUE THREAD

**Issue by turulix** [Ellypse/IntelliJ-IDEA-Lua-IDE-WoW-API#29]
I'm not sure if this repository is still active (last update 8 months ago, and the owner hasn't commited something within years).

For anyone looking for an uptodate version that gets updated nightly, i made a fork:
https://github.com/turulix/IntelliJ-IDEA-Lua-IDE-WoW-API

**Comment by brittyazel** [comment]
Yeah, sorry I've been using Ketho's [vscode wow API ](https://github.com/Ketho/vscode-wow-api) that works pretty great with EmmyLua and IntelliJ. That's why I've stoped being active here 😐
